### PR TITLE
update deployment window picture + upload-launch doc

### PR DESCRIPTION
**Pull request by paulbalaji**
#### Description
- update deployment window picture
- update `upload-launch` docs

#### Tests
- [x] improbadoc local
- [x] techwriter approval

#### Documentation
this _is_ the documentation for the following FPS PR: https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/116

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

**Comment by jamiebrynes7** [comment]
Keep in mind this doc change is for the next release - meaning once this is merged we cannot republish. 

I would sync up with @oblm as he's been making quite a few docs changes recently

**Comment by paulbalaji** [comment]
Putting this PR on hold until all current to-be-published changes/PRs are merged/rolled out.

File: docs/content/get-started/upload-launch.md [hunk 1]
```diff
@@ -70,15 +70,17 @@ Uploaded assembly my_assembly to project unity_gdk successfully.
...
 #### 3. Launch a cloud deployment
 
 The next step is to [start a cloud deployment](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud#5-deploy-the-project) using the assembly that you just uploaded. When you launch a cloud deployment you have to provide four parameters:
 
-* **Assembly Name** - this identifies the worker assemblies to use. 
+* **Assembly Name** - this identifies the worker assemblies to use.
 * **Deployment Name** - this labels your deployment in the SpatialOS Console.
 * **Snapshot File** - this specifies which snapshot file the deployment uses. A snapshot declares the state of the world as the deployment starts.
 * **Launch Config File** - this is the deployment's launch configuration `.json` file which declares the world and load balancing configuration.
+* **Deployment Region** - the region that your deployment will be started in, set to `US` by default.
```
Comment (ElleEss) [inline docs/content/get-started/upload-launch.md]:
```suggestion
* **Deployment Region** - the geographic location that your deployment starts in: `EU` for Europe or `US` for USA. Set to `US` by default.
```
Make it clear this is geography and the real-world as we talk about worlds a lot and region could mean region of the SpatialOS world or your game world.
Also made it into active voice and present simple so the text is stronger and less convoluted.

File: docs/content/get-started/upload-launch.md [hunk 2]
```diff
@@ -92,6 +94,7 @@ To launch your cloud deployment:
 |**Deployment Name**| A name of your choice - for example; `shootyshooty`. <br/>(The name needs to conform to the following regex: `[a-z0-9_]{2,32}`.)|
 |**Snapshot File**|`snapshots/cloud.snapshot`<br/>Note that this is a relative path from the root of your SpatialOS project.|
 |**Launch Config File** | `cloud_launch_large.json`<br/>Note that this is a relative path from the root of your SpatialOS project.|
+|**Deployment Region**|This can be set to `US` (default) or `EU`. Select the region closest to you for lower latencies when connecting clients. Note that the simulated players deployment will also be deployed to the same region.|
```
Comment (ElleEss) [inline docs/content/get-started/upload-launch.md]:
```suggestion
|**Deployment Region**| You can set this to `US` for the USA (default) or `EU` for Europe. Select the geographical location closest to you for lower latencies when connecting clients. <br/> (Note that the simulated players deployment deploys to the same region - you will find out more about simulated players in later steps.)|
```

File: docs/content/get-started/upload-launch.md [hunk 3]
```diff
@@ -70,15 +70,17 @@ Uploaded assembly my_assembly to project unity_gdk successfully.
 
 > **TIP:** Depending on your network speed, the upload may take a little while (1-10 minutes) to complete.
 
+<br/>
 
 #### 3. Launch a cloud deployment
 
 The next step is to [start a cloud deployment](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud#5-deploy-the-project) using the assembly that you just uploaded. When you launch a cloud deployment you have to provide four parameters:
```
Comment (ElleEss) [inline docs/content/get-started/upload-launch.md]:
```suggestion
The next step is to [start a cloud deployment](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud#5-deploy-the-project) using the assembly that you just uploaded. When you launch a cloud deployment you have to provide five parameters:
```

File: docs/content/get-started/upload-launch.md [hunk 4]
```diff
@@ -92,6 +94,7 @@ To launch your cloud deployment:
 |**Deployment Name**| A name of your choice - for example; `shootyshooty`. <br/>(The name needs to conform to the following regex: `[a-z0-9_]{2,32}`.)|
 |**Snapshot File**|`snapshots/cloud.snapshot`<br/>Note that this is a relative path from the root of your SpatialOS project.|
 |**Launch Config File** | `cloud_launch_large.json`<br/>Note that this is a relative path from the root of your SpatialOS project.|
+|**Deployment Region**|This can be set to `US` (default) or `EU`. Select the region closest to you for lower latencies when connecting clients. Note that the simulated players deployment will also be deployed to the same region.|
 |**Enable Simulated Players**| Ensure this is checked.|
 |**Override Name**| Leave this unchecked.<br/>This allows you to change the name of the simulated players deployment.|
```
Comment (ElleEss) [inline docs/content/get-started/upload-launch.md]:
```suggestion
|**Override Name**| Leave this unchecked.<br/>This allows you to change the name of the simulated players deployment - more about simulated players in the next steps.|
```

File: docs/content/get-started/upload-launch.md [hunk 5]
```diff
@@ -106,7 +109,9 @@ To launch your cloud deployment:
 **Note:** You will see that there are two Deployment Names and two Launch Config Files listed in the Deployment window; the first listed is your FPS Starter Project, the second is a deployment for simulated players. You will find out more about simulated players in later steps.
 <br/>
 
-**It's finished when:** You see two messages printed in your Unity Console indicating both deployments have been started:
+**It's finished when:** the Console pages for both deployments have opened up in your browser.
```
Comment (ElleEss) [inline docs/content/get-started/upload-launch.md]:
```suggestion
**It's finished when:** The Console pages for both deployments open automatically in your browser.
```
Make it active voice and clear the user doesn't have to do anything.

File: docs/content/get-started/upload-launch.md [hunk 6]
```diff
@@ -106,7 +109,9 @@ To launch your cloud deployment:
...
+
+You should also see two messages printed in your Unity Console indicating both deployments have been started:
```
Comment (ElleEss) [inline docs/content/get-started/upload-launch.md]:
```suggestion
You should also see two messages printed in your Unity Console indicating both deployments have started:
```
Make this active voice.

**Pull request merged**